com.github.SuperTuxAdvance.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "com.github.SuperTuxAdvance",
```

Comment:
this app-id is invalid. github based are never `com.github` as per the documentation. and are made of 4 components.